edx_rest_framework_extensions/config.py

```diff
@@ -2,4 +2,6 @@
 Application configuration constants and code.
 """
 
-SWITCH_ENFORCE_JWT_SCOPES = 'oauth2.enforce_jwt_scopes'
+NAMESPACE_SWITCH = 'oauth2'
```

Comment:
I think the before code is simpler.  Not sure why this is needed.

Reply:
The edx-platform imports the switch from here and needs the namespace value to use waffle_utils.